input/data/2014/2014A%26A...562A..40H/tev-000117.yaml

```diff
@@ -1,4 +1,23 @@
 source_id: 117
 reference_id: 2014A&A...562A..40H
 
-# TODO: add data
+data:
+  livetime: 120
+  significance: 8.2
+  excess: 117
+
+pos:
+  ra: {val: 18h18m4.8s, err: 3.1s, err_sys: }
+  dec: {val: -15d28m1s, err: 43s, err_sys: }
```

Comment:
The paper states 
> In addi- tion to the statistical uncertainty of this fit, there is also a system- atic uncertainty due to the pointing precision of the telescope ar- ray of about 20′′

so you could add that info for err_sys = 20 arcsec for both axes